### ISSUE TITLE
Basic printing window doesn't display properly on OSX

### ISSUE THREAD

**Issue by mrjohnc** [daid/LegacyCura#818]
![screen shot 2014-04-14 at 19 33 09](https://cloud.githubusercontent.com/assets/7293392/2699164/8b7d16fc-c403-11e3-9793-f6fb8996587b.png)
The basic printing window on OSX Lion on a Macbook (white 2010) doesn't display properly, the temperatures are stacked underneath the buttons. This doesn't affect the function and the buttons still work, just cosmetic (and kind of hard to read the numbers)

**Comment by daid** [comment]
Added some changes to the RC4:
http://software.ultimaker.com/Cura_closed_beta/
Which might help in preventing this issue.

**Comment by mrjohnc** [comment]
Thanks Daid!

**Comment by trikko** [comment]
Still wrong for me (in a different way!)
![bug](https://cloud.githubusercontent.com/assets/647157/3098405/a53245e0-e5eb-11e3-8363-1af65b4b9a72.png)

**Comment by daid** [comment]
No, that's all intentional. (There is an invisible pause button between the print and the cancel)

The extra spacing is to make sure all text fits on there. It's a quick workaround, but it's better then before as no text overlaps with other stuff.
